### ISSUE TITLE
[BUG] E-Mail regex is incorrect

### ISSUE THREAD

**Issue by Myzel394** [bunkerity/bunkerized-nginx#170]
**Description**
The email regex for `EMAIL_LETS_ENCRYPT` is incorrect. There are emails available with `_`.

**How to reproduce**
Use an email with a `_` in it. E.g `EMAIL_LETS_ENCRYPT=test_test@protonmail.com`

**Logs**
```
Ignoring EMAIL_LETS_ENCRYPT=my_email_address@protonmail.com (doesn't match regex : ^([a-z0-9\-\.]+@[a-z\-0-9\.]+|.{0})$)
```

**Suggestion**
Use
```
^([a-z0-9\-\._]+@[a-z\-0-9\.]+|.{0})$
```

**Comment by fl0ppy-d1sk** [comment]
Hey @Myzel394,

This is not a very common use case IMO but you're right, the regex is missing some "special" characters as shown in the (RFC 3696](https://datatracker.ietf.org/doc/html/rfc3696#section-3). Unless we have some critical issues to address in the v1.2.X version, I would suggest to include the fix into the next 1.3.X release.

**Comment by Myzel394** [comment]
Okay thank you, I think I can wait until 1.3 :D

**Comment by fl0ppy-d1sk** [comment]
Hello @Myzel394,

It should be fixed in the [v1.3.0 release](https://github.com/bunkerity/bunkerized-nginx/releases/tag/v1.3.0).
